api/test/docker/apisix_config.yaml

```diff
@@ -20,8 +20,6 @@
 etcd:
   host:
     - "http://172.16.238.10:2379"
-    - "http://172.16.238.11:2379"
-    - "http://172.16.238.12:2379"
```

Comment:
We should keep them, people generally use etcd in a cluster instead of a single node

Reply:
OK, I will modify to keep it. But currently we don't have test cases that actually use it, such as skipping failed nodes, auto reconnect, etc.